torch/_inductor/kernel/conv.py

```diff
@@ -335,10 +338,23 @@ def convolution(
 
     x.realize()
     weight.realize()
-    layout = conv_layout(x, weight, None, **kwargs)
-    req_stride_order = ir.get_stride_order(V.graph.sizevars.size_hints(layout.stride))
-    x = ir.ExternKernel.require_stride_order(x, req_stride_order)
-    weight = ir.ExternKernel.require_stride_order(weight, req_stride_order)
+
+    # ndim can be 1 for convolution in models such as demucs
+    # TODO: check if channels last can benefit Conv1d
```

Comment:
There's no channels_last1d format, but for Conv1d I think we can just rewrite it as a conv2d.

Reply:
Do we rewrite by adding a size=1 dimension for input and weight?

Reply:
Yeah.

Reply:
I changed the TODO based on discussion here.